src/components/main/index.tsx

```diff
@@ -283,10 +319,14 @@ export default function MainPage({ title, favicon, currentInfos, signature }: Ma
                     <MembershipListPage memberships={dummyMemberships} />
                 </TabPanel>
                 <TabPanel value={value} index={1}>
-                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />}>
+                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />} onClick={handleClickOpen}>
                         Full Screen
                     </Button>
-                    <HistoryListPage />
+                    <Dialog open={open} fullScreen onClose={handleClose} TransitionComponent={Transition}>
+                        <DialogHeader onClose={handleClose} />
+                        <HistoryListPage histories={dummyHistoryLogs} />
+                    </Dialog>
+                    <HistoryListPage histories={dummyHistoryLogs} />
```

Comment:
Full screen appearance. 
<img width="1918" alt="스크린샷 2020-09-03 오전 2 30 58" src="https://user-images.githubusercontent.com/14817195/92016578-8245b700-ed8d-11ea-809f-5bbdde033fbe.png">

Reply:
It's great!